recipe.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Healthy Protein Pancakes</title>
+</head>
+<body>
+    <h2><Healthy Protein Pancake/h2>
+    <img
+    src= "https://www.erinliveswhole.com/wp-content/uploads/2021/04/proteinpancakes-5.jpg"
+    height = "300px"
+    />
+    <div>
+        ---------------------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
there is an element for this

Reply:
Angie I used the <hr> but the test still have not been able to adjust to length of 4